var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -78,12 +93,42 @@ class NaluWind(CMakePackage, CudaPackage):
             "hypre@develop +mpi+cuda~int64~superlu-dist cuda_arch={0}".format(_arch),
             when="+hypre+cuda cuda_arch={0}".format(_arch),
         )
+    for _arch in ROCmPackage.amdgpu_targets:
+        depends_on(
+            "trilinos@13.4.0.2022.10.27: "
+            "~shared+exodus+tpetra+zoltan+stk+boost~superlu-dist~superlu"
+            "+hdf5+shards~hypre+gtest+rocm amdgpu_target={0}".format(_arch),
+            when="+rocm amdgpu_target={0}".format(_arch),
+        )
+        depends_on(
+            "hypre+rocm amdgpu_target={0}".format(_arch),
+            when="+hypre+rocm amdgpu_target={0}".format(_arch),
+        )
+
     depends_on("trilinos-catalyst-ioss-adapter", when="+catalyst")
     depends_on("fftw+mpi", when="+fftw")
     depends_on("nccmp")
     # indirect dependency needed to make original concretizer work
     depends_on("netcdf-c+parallel-netcdf")
     depends_on("boost +filesystem +iostreams cxxstd=14", when="+boost")
+    cxxstd = ["17"]
```

Comment:
Maybe use a name which is less likely to clash (this will be defined at the class level)